### PR TITLE
Start creating SRA

### PR DESCRIPTION
**Pull request by achim3**
### Description of Change

- update EDT functionality

### Test results

- All OK

### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

**Review by achim3 — COMMENTED** [review]
Fine

**Comment by Svedapete** [comment]
👍

**Comment by achim3** [comment]
Please review

**Review by achim3 — COMMENTED** [review]
review

File: SRA.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# current
+## requested change
```
Comment (Svedapete) [inline SRA.md:2]: jksjdfjs

**Review by Svedapete — APPROVED** [review]

File: SRA.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# current
```
Comment (achim3) [inline SRA.md:1]: tetst

**Pull request merged**